http/server.go

```diff
@@ -61,7 +61,7 @@ type Server struct {
 }
 
 // NewServer creates a new server
-func NewServer(config *frames.Config, addr string, logger logger.Logger, historyServer *utils.HistoryServer) (*Server, error) {
+func NewServer(config *frames.Config, addr string, logger logger.Logger, historyServer *utils.HistoryServer, version string) (*Server, error) {
```

Comment:
If you need to have this in both "servers" maybe you can just have this statically here, meaning just dont go through the `api` when requesting version